elements/phaseindicator.lua

```diff
@@ -75,21 +76,24 @@ local function Update(self, event, unit)
 		element:PreUpdate()
 	end
 
-	local isInSamePhase = not UnitPhaseReason(unit)
-	if(not isInSamePhase and UnitIsPlayer(unit) and UnitIsConnected(unit)) then
+	local phaseReason = UnitIsConnected(unit) and UnitPhaseReason(unit) or nil
```

Comment:
Why the `or nil` at the end here?

Reply:
For the sake of consistency, tbhon, that's one of `UnitPhaseReason`'s expected returns.